packages/form-footer/src/FormFooter.tsx

```diff
@@ -130,6 +179,8 @@ export interface FormFooterProps extends HTMLElementProps<'footer'> {
  */
 export default function FormFooter({
   primaryButton,
+  cancelButtonProps,
+  backButtonProps,
   onCancel,
   cancelButtonText = 'Cancel',
```

Comment:
Ran into some issues with how we handle hiding the cancel button

I would expect that not defining `cancelButtonText` would lead to NOT rendering the button similar to the approach for the back button. However, it seems that if left undefined, the text will default to `'Cancel'`. To not render the cancel button, `cancelButtonText` must be an empty string

This seems to create some issues with maintaining backwards compatibility with these newly added props. In order to maintain backwards compatibility, I think we should keep these defaults as-is

Reply:
I believe the intent here was to render a default cancel button unless otherwise specified. That definitely results in some interesting backwards compat issues though with the new props